stdlib/timezone/timezone.flux

```diff
@@ -0,0 +1,45 @@
+// Package timezone defines functions for setting timezones
+// on the location option in package universe.
+package timezone
+
+
+// utc is the default location with a completely linear clock
+// and no offset. It is used as the default for location related
+// options.
+utc = {zone: "UTC", offset: 0h}
+
+// fixed is a function that constructs a location with a fixed offset.
+//
+// ## Parameters
+// - `offset` is the fixed duration for the location offset.
+//   This duration is the opposite of the offset for the `window()` function.
```

Comment:
I think it would be good to define the offset as the timezone offset from UTC instead of defining it as the opposite of window's offset. 

We can still call out the behavior that it has the opposite behavior of window's offset but I don't think we should define it that way.